CSV/sample-entries.csv

```diff
@@ -20,9 +20,12 @@ dtse,"DTS Express low bit rate audio, also known as DTS LBR",Audio,DTS,0xAC
 dtsh,DTS-HD High Resolution Audio,Audio,DTS,0xAA
 dtsl,DTS-HD Master Audio,Audio,DTS,0xAB
 dtsx,DTS:X,Audio,DTS,
-dvav,AVC-based “Dolby Vision”,Video,Dolby Vision,
-dvhe,HEVC-based “Dolby Vision”,Video,Dolby Vision,
+dva1,AVC-based “Dolby Vision” derived from 'avc1',Video,Dolby Vision,
+dvav,AVC-based “Dolby Vision” derived from 'avc3',Video,Dolby Vision,
+dvh1,HEVC-based “Dolby Vision” derived from 'hvc1',Video,Dolby Vision,
+dvhe,HEVC-based “Dolby Vision” derived from 'hev1',Video,Dolby Vision,
 ec-3,Enhanced AC-3 audio,Audio,ETSI AC-3,0xA6
+ec+3,Enhanced AC-3 audio with JOC,Audio,ETSI E-AC-3+JOC,
```

Comment:
The last value should also match the `linkname` you used in the specification.csv: etsi-eac3+